dock/util.py

```diff
@@ -176,19 +176,27 @@ def structure(self):
             {"instruction": "FROM",       # always upper-case
              "startline": 0,              # 0-based
              "endline": 0,                # 0-based
-             "content": "From fedora\n"},
+             "content": "From fedora\n",
+             "value": "fedora"},
 
             {"instruction": "CMD",
              "startline": 1,
              "endline": 2,
-             "content": "CMD yum -y update && \\\n    yum clean all\n"}
+             "content": "CMD yum -y update && \\\n    yum clean all\n",
+             "value": "yum -y update && yum clean all"}
```

Comment:
What would the value be expected to be for `"CMD yum -y update &&     \\\n  yum clean all"`? Currently leading whitespace is stripped while whitespace up to `\` is preserved. Is that what docker does? What about `"RUN ls&&\\\nusb\n"`?